gammapy/spectrum/models.py

```diff
@@ -26,6 +27,110 @@ def __call__(self, energy):
         """Call evaluate method of derived classes"""
         return self.evaluate(energy, **self.parameters)
 
+    def __str__(self):
+        """String representation"""
+        ss = self.__class__.__name__
+        for parname, parval in self.parameters.items():
+            ss += '\n{parname} : {parval:.3g}'.format(**locals())
+        return ss
+
+    def to_sherpa(self, name='default'):
+        """Return `~sherpa.models.ArithmeticModel`
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the sherpa model instance
+        """
+        import sherpa.models as m
+        if isinstance(self, PowerLaw):
```

Comment:
I think this should be implemented in `PowerLaw.to_sherpa`, not the base class, no?
(The base class could just not have this method. Or it could have one that raises `NotImplementedError`?)